Protocol/Protocol.csproj

```diff
@@ -48,8 +48,10 @@
     <ItemGroup>
         <InternalsVisibleTo Include="$(MSBuildProjectName)Tests" />
 
-        <!-- Needed for DIS -->
+        <!-- Needed for DIS unit tests -->
         <InternalsVisibleTo Include="SLDisUnitTestsShared" />
```

Comment:
if this was added only because of these internal ctors:

internal Protocol(Read.IProtocol read, EditXml.XmlElement editNode) : base(read, null, editNode)
internal Protocol(Read.IProtocol read) : base(read, null, CreateEditNode(read))

we could just make these public instead

Reply:
Changed the code so that it doesn't need the internal ctors anymore. Removed the InternalsVisibleTo for DIS & Validator